pkg/comp-functions/functions/common/instance_namespace.go

```diff
@@ -101,7 +101,9 @@ func createInstanceNamespace(ctx context.Context, serviceName, compName, claimNa
 
 	disabled, err := isBillingDisabled(controlNS, instanceNamespace, compName, svc)
 	if err != nil {
-		return err
+		// we don't return here, otherwise we risk the namespace getting deleted
```

Comment:
can You further explain? so I understand whole picture in detail

Reply:
Was the explanation during the daily on Friday enough?

Reply:
yes of course